src/Compilers/Core/Portable/DiagnosticAnalyzer/CompilerDiagnosticAnalyzer.cs

```diff
@@ -16,7 +16,9 @@ internal abstract partial class CompilerDiagnosticAnalyzer : DiagnosticAnalyzer
         private ImmutableArray<DiagnosticDescriptor> _supportedDiagnostics;
 
         protected abstract CommonMessageProvider MessageProvider { get; }
-        protected abstract ImmutableArray<int> GetSupportedErrorCodes();
+
+        // internal as this is called from tests
+        internal abstract ImmutableArray<int> GetSupportedErrorCodes();
```

Comment:
💡 It would be better to use a TestAccessor to access this without increasing accessibility.

Reply:
don't care for now.  just doc'ing :)